### ISSUE TITLE
In EPUB Accessibility 1.1, key words such as "SHOULD" ought to be written in all uppercase consistently when used in their normative sense.

### ISSUE THREAD

**Issue by jasonjgw** [w3c/epub-specs#1802]
I noticed, particularly in normative subsections of section 3.4.2, that "SHOULD" is not capitalized consistently when used in its normative sense. Moreover, there is no other indication in the text of which objectives are required and which are optional.

I think all uppercase should be used consistently when the key words defined in section 1.5 appear in normative sections of the document.

**Comment by mattgarrish** [comment]
Yes, those were lowercase earlier on in the revision when the whole section was proposed as informative. Since compliance is what is optional, not the normative requirements, they should have been flipped back to uppercase.
